Examples/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(PcapPlusPlusExamples)
+
+set(CMAKE_PROJECT_HOMEPAGE_URL "https://pcapplusplus.github.io/")
+
+if(TARGET Pcap++)
+  # Are we building from the PcapPlusPlus project ??
+  message(STATUS "Building Example from PcapPlusPlus project")
+  # Alias the project to make it compatible with imported package
+  add_library(PcapPlusPlus::Pcap++ ALIAS Pcap++)
+else()
+  find_package(PcapPlusPlus CONFIG REQUIRED)
+  message(STATUS "Using Pcap++ ${PcapPlusPlus_VERSION}")
+  message(STATUS "Include dir: ${PcapPlusPlus_INCLUDE_DIR}")
+endif()
+
+# Set C++11
+set(CMAKE_CXX_STANDARD 11)
+# popen()/pclose() are not C++ standards
+set(CMAKE_CXX_EXTENSIONS ON)
+
+add_subdirectory(Arping)
+add_subdirectory(ArpSpoofing)
+add_subdirectory(DNSResolver)
+add_subdirectory(DnsSpoofing)
+add_subdirectory(HttpAnalyzer)
+add_subdirectory(IcmpFileTransfer)
+add_subdirectory(IPDefragUtil)
+add_subdirectory(IPFragUtil)
+add_subdirectory(PcapPlusPlus-benchmark)
+add_subdirectory(PcapPrinter)
+add_subdirectory(PcapSearch)
+add_subdirectory(PcapSplitter)
+add_subdirectory(SSLAnalyzer)
+add_subdirectory(TcpReassembly)
+add_subdirectory(TLSFingerprinting)
+
+add_subdirectory(Tutorials)
```

Comment:
I'm not sure if we should build the tutorials by default, but I don't have a strong opinion. What do you think? 🤔

Reply:
Agree, I will add an option for this